config/initializers/middleware.rb

```diff
@@ -14,7 +15,13 @@
 if Rails.application.config.action_controller.perform_caching
   # Serve the static page cache
   Rails.application.config.action_controller.page_cache_directory = Rails.root.join("public/cached_pages")
+
   Rails.application.config.middleware.insert_before \
     ActionDispatch::Static, ActionDispatch::Static, File.join(Rails.application.config.root, "public", "cached_pages"),
     headers: { "Cache-Control" => "max-age=#{5.minutes.to_i}, public, immutable" }
+
+  # Send request event to dfe analytics if static page is served
+  Rails.application.config.middleware.insert_before \
+    ActionDispatch::Static, Middleware::DfeAnalytics, File.join(Rails.application.config.root, "public", "cached_pages"),
```

Comment:
Not sure the headers are needed here; from memory we're using the headers in the page cache middleware to tell it to serve content with a cache-control of 5 minutes. I'm _fairly_ certain that under the hood it caches the page indefinitely (until a deployment essentially clears the cache/files).